### PR TITLE
Examples documentation cleanup

### PR DESCRIPTION
**Pull request by ocofaigh**
### Description

Examples documentation cleanup

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by SirSpidey — CHANGES_REQUESTED** [review]
See whether these suggestions work. A lot of it is expanding abbreviations and using official product names. We'll want to use the same language across all the examples.

File: examples/replication/README.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+
+Resources provisioned by this example:
+- A new resource group, if existing one is not passed in.
+- One target and one source Cloud Object Storage instance in the given resource group and region.
+- One target and one source COS bucket.
 - An IAM authorization policy to let the source instance write to the target bucket.
-- A rule to copy everything from the source bucket to the target bucket.
+- A replication rule to copy everything from the source bucket to the target bucket.
+
+- A Sysdig and Activity Tracker instance (if existing one is not passed in) in the given resource group and region.
```
[the remark this answers is not in the document]
Comment (ocofaigh) [reply examples/replication/README.md:13]: The example only supports passing in existing AT instance (not sysdig) - Can you suggest a reword for that @SirSpidey ?

File: examples/replication/README.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+- A Cloud Object Storage instance in the given resource group and region.
```
[the remark this answers is not in the document]
Comment (ocofaigh) [reply examples/replication/README.md:15]: actually these bullet points were not supposed to here :) removed now

File: examples/replication/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ocofaigh) [reply examples/replication/README.md]: Going to split into 2 separate bullet points

[the remark this answers is not in the document]
Comment (ocofaigh) [reply examples/replication/README.md:14]: Going to split into 2 separate bullet points

**Review by SirSpidey — APPROVED** [review]
let's go and see how it looks now...

**Comment by ocofaigh** [comment]
Holding off on merge until https://github.com/terraform-ibm-modules/terraform-ibm-cos/pull/460 is merged into this, and we finalise a diagram

**Review by SirSpidey — APPROVED** [review]
Approving because I think we've nailed the various updates for now.

**Review by SirSpidey — APPROVED** [review]
:shipit:

**Pull request merged**